### PR TITLE
Removed 4 apps and added 1 app.

### PR DESCRIPTION
**Pull request by invinciblevenom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,7 +209,6 @@ Table of Contents
 - [**Emerald Launcher**](https://github.com/HenriDellal/emerald) <sup>**[[F-Droid](https://f-droid.org/app/ru.henridellal.emerald)]**</sup>
 - [**Rootless Pixel Launcher**](https://github.com/amirzaidi/Launcher3) <sup>**[[F-Droid](https://f-droid.org/app/amirz.rootless.nexuslauncher)]**</sup>
 - [**BlissLauncher**](https://gitlab.e.foundation/e/apps/BlissLauncher) <sup>**[[F-Droid](https://f-droid.org/app/foundation.e.blisslauncher)]**</sup>
-- [**Posidon Launcher**](https://github.com/leoxshn/posidonLauncher) <sup>**[[F-Droid](https://f-droid.org/app/posidon.launcher)]**</sup>
```
Comment (offa) [inline README.md:214]: What a pity, it was really promising. While the website is still up and the last release wasn't too long ago, I couldn't find any infos about what happened to the project unfortunately. @leoxshn seems still active on GH though.

**Comment by offa** [comment]
Thank you! 👍

File: README.md (hunk 1 above)
Comment (invinciblevenom) [reply README.md:214]: Yeah! It was a good launcher though. Will see if anyone forks and maintains it

**Comment by invinciblevenom** [comment]
> 
> 
> Thank you! 👍

Welcome!

**Pull request merged**